#!/usr/bin/python3


def raise_exception_msg(mesage=""):
    raise NameError(err)
